language/move-vm/runtime/src/loader.rs

```diff
@@ -631,23 +659,15 @@ impl Loader {
         verify_no_missing_modules: bool,
         log_context: &impl LogContext,
     ) -> VMResult<()> {
-        DuplicationChecker::verify_module(&module)?;
-        SignatureChecker::verify_module(&module)?;
-        InstructionConsistency::verify_module(&module)?;
-        ability_field_requirements::verify_module(&module)?;
-        constants::verify_module(&module)?;
-        RecursiveStructDefChecker::verify_module(&module)?;
-        InstantiationLoopChecker::verify_module(&module)?;
-        CodeUnitVerifier::verify_module(&module)?;
```

Comment:
Here is the loader change to drop each individual checks and keep it in-sync with the checks in `bytecode_verifier::verify_module()`